addons/account/views/account_journal_dashboard_view.xml

```diff
@@ -259,7 +259,7 @@
                         </div>
                         <!-- On the right, show other common informations/actions -->
                         <div id="dashboard_bank_cash_right" class="col-12 col-sm-7 o_kanban_primary_right">
-                            <div class="row" t-if="dashboard.nb_lines_bank_account_balance > 0">
+                            <div class="row" t-if="dashboard.has_at_least_one_statement and dashboard.nb_lines_bank_account_balance > 0">
```

Comment:
I don't think that fix would be ok.
With bank synchronization, you don't need to have any statement.
The statement is just a convenient stuff for allowing people to make their bank entry the way they're used to.